src/templates/base_register.html

```diff
@@ -79,15 +78,14 @@
     })
 });
 </script>
-
-{% endblock %}
-{% block extrajs %}
-{% endblock %}
```

Comment:
I send PR for helper.py. Not for others. When I sent PR, previous unwanted commits added automatically.